core/domain/app_feedback_report_validators_test.py

```diff
@@ -274,16 +275,17 @@ def test_model_with_web_schema_version_greater_than_current_schema_fails(
         web_entity.update_timestamps()
         web_entity.put()
 
-        self.expected_validation_output.append((
-            u'[u\'failed validation check for report schema version check of '
-            'AppFeedbackReportModel\', '
-            '[u\'Entity id %s: web report schema version %s is outside the '
-            'range of supported versions [%s, %s]\']]') % (
-                web_entity.id, web_entity.web_report_info_schema_version,
-                feconf.MINIMUM_WEB_REPORT_SCHEMA_VERSION,
-                feconf.CURRENT_WEB_REPORT_SCHEMA_VERSION))
+        self.expected_successful_validation_output.append(
```

Comment:
Ah, this is a bit odd. I think this might be where we were crossing paths on the previous discussion thread. We should define a new variable here for expected output. It could be based on the successful output, but it doesn't make sense to redefine success to include errors.

Reply:
Yes I was thinking this exactly, but then I was also on the fence about adding a temporary variable solely for naming conventions. I'll add it since it then